commit f2a7721708e1e40e06d9ecc5a2a590af5098d9ad (refs/changes/06/5106/2)
Author: Trent Mick <trentm@gmail.com>
Date:   2018-11-22T13:55:31-08:00 (11 months ago)
    
    MANTA-4030 manta images broken by transitive lru-cache update
    Reviewed by: Josh Wilsdon <josh@wilsdon.ca>
    Approved by: Josh Wilsdon <josh@wilsdon.ca>

diff --git a/package.json b/package.json
index 1aa8c7f..ac8822d 100644
--- a/package.json
+++ b/package.json
@@ -1,17 +1,18 @@
 {
   "name": "binder",
-  "description": "SmartDataService DNS Service",
-  "version": "1.3.0",
+  "description": "Triton Data Center internal DNS service",
+  "version": "1.3.1",
   "author": "Joyent (joyent.com)",
   "license": "MPL-2.0",
   "private": true,
+  "// lru-cache": "attempting to pin version while satisfying transitive deps, because lru-cache@4.1.4 includes yallist@3 which breaks node --abort-on-uncaught-exception with node <v4",
   "dependencies": {
     "artedi": "1.3.0",
     "assert-plus": "0.1.5",
     "bunyan": "1.5.1",
     "clone": "0.1.1",
-    "dtrace-provider": "0.8.6",
-    "lru-cache": "2.7.0",
+    "dtrace-provider": "0.8.7",
+    "lru-cache": "4.1.3",
     "mname": "1.5.0",
     "mooremachine": "^2.1.0",
     "mname-client": "0.6.0",
@@ -25,7 +26,7 @@
     "zkstream": "0.11.0"
   },
   "engines": {
-    "node": ">=0.8"
+    "node": ">=0.10"
   },
   "devDependencies": {
     "expiring-lru-cache": "2.1.0",
